CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
   - Binaries renamed from `oauth2_proxy` to `oauth2-proxy`
 - [#432](https://github.com/oauth2-proxy/oauth2-proxy/pull/432) Update ruby dependencies for documentation (@theobarberbany)
 - [#471](https://github.com/oauth2-proxy/oauth2-proxy/pull/471) Add logging in case of invalid redirects (@gargath)
-- [#462](https://github.com/oauth2-proxy/oauth2-proxy/pull/462) Allow HTML in banner message (@eritikass).
+- [#413](https://github.com/oauth2-proxy/oauth2-proxy/pull/413) Add -set-basic-auth param to set the Basic Authorization header for upstreams (@morarucostel).
```

Comment:
Something went wrong here, need a new entry rather than overwriting the existing one

Reply:
I don't know what happend, I don't recall overwriting an existing entry...
However I introduce re-introduced the previous one, so hopefully it fixed the CHANGELOG issue.